ontology/EBUCore/ebucore.owl

```diff
@@ -5181,6 +5180,31 @@ ec:Asset rdf:type owl:Class ;
                            owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
                            owl:onClass ec:DateTimeObject
                          ] ,
+                         [ rdf:type owl:Restriction ;
+                           owl:onProperty ec:dateIssued ;
+                           owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                           owl:onClass ec:DateTimeObject
+                         ] ,
+                         [ rdf:type owl:Restriction ;
+                           owl:onProperty ec:dateModified ;
+                           owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                           owl:onClass ec:DateTimeObject
+                         ] ,
+                         [ rdf:type owl:Restriction ;
+                           owl:onProperty ec:dateReleased ;
+                           owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                           owl:onClass ec:DateTimeObject
+                         ] ,
+                         [ rdf:type owl:Restriction ;
+                           owl:onProperty ec:datelicenseEnd ;
```

Comment:
Typo: "datelicenseEnd" should be camel cased and read "dateLicenseEnd"

Reply:
I'll raise an issue.